test-app/src/widgets/AppWindow.js

```diff
@@ -852,6 +958,7 @@ export default class AppWindow extends createjs.Container {
       height: 60,
       width: 250,
     };
+    let reset;
```

Comment:
Just trying to understand, why was this changed from using const to using let?

Reply:
This change was done to resolve a listing error. Since `reset` is been used on lines 973 & 1005 before it's been actually created/initialized hence a listing error was thrown.